WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingView.swift

```diff
@@ -5,53 +5,69 @@ struct StoreOnboardingView: View {
     private let viewModel: StoreOnboardingViewModel
     private let taskTapped: (StoreOnboardingTask) -> Void
     private let viewAllTapped: (() -> Void)?
+    private let dismissAction: (() -> Void)?
 
     init(viewModel: StoreOnboardingViewModel,
          taskTapped: @escaping (StoreOnboardingTask) -> Void,
-         viewAllTapped: (() -> Void)? = nil) {
+         viewAllTapped: (() -> Void)? = nil,
+         dismissAction: (() -> Void)? = nil) {
         self.viewModel = viewModel
         self.taskTapped = taskTapped
         self.viewAllTapped = viewAllTapped
+        self.dismissAction = dismissAction
     }
 
     var body: some View {
-        VStack(alignment: viewModel.isExpanded ? .center : .leading, spacing: Layout.verticalSpacing) {
-            // Progress view
-            StoreSetupProgressView(isExpanded: viewModel.isExpanded,
-                                   totalNumberOfTasks: viewModel.taskViewModels.count,
-                                   numberOfTasksCompleted: viewModel.numberOfTasksCompleted)
-
-            // Task list
-            VStack(alignment: .leading, spacing: Layout.verticalSpacingBetweenTasks) {
-                ForEach(Array(viewModel.taskViewModels.enumerated()), id: \.offset) { index, taskViewModel in
-                    let isLastTask = index == viewModel.taskViewModels.count - 1
-
-                    StoreOnboardingTaskView(viewModel: taskViewModel,
-                                            showDivider: !isLastTask) { task in
-                        taskTapped(task)
+        VStack {
+            Color(.systemColor(.systemGray6))
```

Comment:
💡 We don't usually use `systemColor`, I'd suggest switching to `.listBackground`:
```suggestion
            Color(uiColor: .listBackground)
```

Reply:
Done in 5787ab7736b5036b2b42aba78707031cbc41cc12